launch/prefix_tf_republisher.launch.yaml

```diff
@@ -0,0 +1,26 @@
+launch:
+- arg:
+    name: namespace
+    default: ""
```

Comment:
Not a fan of making a namespace an arg since you can just start the whole launch file in a namespace.

Reply:
The respective PR is not yet merged, so I would prefer to leave this in for easier use. Also it's not really a problem.